ansible/inventories/devnet-12/host_vars/prysm-besu-1-arm.yaml

```diff
@@ -0,0 +1,3 @@
+prysm_container_image: gcr.io/prylabs-dev/prysm/beacon-chain:devnet-10
```

Comment:
remove? wrong image imo

Reply:
Nono, I wanna keep this around, to ensure that I remember. 

Its using a different image than our other prysm nodes, as its for ARM. We can leave it around, as we won't be spinning up an arm machine for prysm this time around, but its a good reminder that it needs an arm img.